File: Searching_Algorithms/C/Linear_Search.c
#include <stdio.h>

int search(int arr[],int n, int x)
{
    int i;
    for (i = 0; i < n; i++)
    {
        if(arr[i] == x) 
            return i;
    }
    return -1;
    
}

int main(int argc, char const *argv[])
{
    int arr[] = {3,12,32,44,332,223,5,77,4,55,98};
    int n = sizeof(arr)/sizeof(arr[1]);
    printf("Please enter a digit that you would like to search\n");
    int x;
    scanf("%d",&x);
    int result = search(arr,n,x);
    (result == -1)?printf("Element is not present")
                  :printf("Element is present in %d position",result);
    return 0;
}

File: Searching_Algorithms/Python/Linear_Search.py
def search(arr,x,n):
    i = 0
    for i in range(n):
        if arr[i] == x:
            return i;
    return -1
arr = [3,12,32,44,332,223,5,77,4,55,98]
x = int(input("Please enter a number that you would like to search"))
ret_val = search(arr,x,len(arr))
if ret_val == -1:
    print("The number does not exist")
else:
    print("The number is present in the {} position".format(ret_val))